targets/TARGET_STM/trng_api.c

```diff
@@ -48,14 +48,19 @@ void trng_init(trng_t *obj)
     /*Select PLLQ output as RNG clock source */
     PeriphClkInitStruct.PeriphClockSelection = RCC_PERIPHCLK_RNG;
     PeriphClkInitStruct.RngClockSelection = RCC_RNGCLKSOURCE_PLL;
-#if defined(DUAL_CORE)
+#elif defined(TARGET_STM32WL)
+    RCC_PeriphCLKInitTypeDef PeriphClkInitStruct;
+    /*Select PLLQ output as RNG clock source*/
+    PeriphClkInitStruct.PeriphClockSelection = RCC_PERIPHCLK_RNG;
+    PeriphClkInitStruct.RngClockSelection = RCC_RNGCLKSOURCE_MSI;
```

Comment:
This should be before #elif defined(TARGET_STM32H7)
because next #if defined(DUAL_CORE) has no sense with WL...

Reply:
Do not take into account previous comment.
Done.